homeassistant/components/knx/light.py

```diff
@@ -101,32 +123,32 @@ def color_temp(self):
         return None
 
     @property
-    def min_mireds(self):
+    def min_mireds(self) -> int:
         """Return the coldest color temp this light supports in mireds."""
         return self._min_mireds
 
     @property
-    def max_mireds(self):
+    def max_mireds(self) -> int:
         """Return the warmest color temp this light supports in mireds."""
         return self._max_mireds
 
     @property
-    def effect_list(self):
+    def effect_list(self) -> Optional[List[str]]:
         """Return the list of supported effects."""
         return None
 
     @property
-    def effect(self):
+    def effect(self) -> Optional[str]:
         """Return the current effect."""
         return None
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return true if light is on."""
-        return self._device.state
+        return bool(self._device.state)
```

Comment:
This is Optional[bool] in xknx. None on init until the first telegram sets a state. ToggleEntity doesn't support this.